### DISCUSSION TITLE
2024/01/06/hetzner-storagebox-to-jotta/

### DISCUSSION THREAD

**Discussion by giscus** [jdeluyck/kcoreorg-website#169] [Blog Comments]
# 2024/01/06/hetzner-storagebox-to-jotta/

A year ago I migrated from Backblaze B2 to Hetzner Storageboxes. That worked fine, but passing over the 5TB border made my offsite storage rather expensive (again). So another round of investigations was called upon.

https://kcore.org/2024/01/06/hetzner-storagebox-to-jotta/

**Comment by magnatebushing0o** [comment]
Is Jottacloud as a backend still working good for you?
I've been using jottacloud for a couple of years. First with duplicati, then with restic/rclone until now. It used to work very well but recently uploads are very slow which is not jottacloud's fault (I think) but somehow restic and rclone do not work well together using the rclone serve restic stdin method.
It always uploads a pack reasonably fast but then waits for seconds or even a minute until the next gets uploaded. Do you have similar issues? And if not what does your rclone command look like that restic uses?

**Reply by magnatebushing0o** [reply]
Nevermind, my TMPDIR was on a slow device. Silly mistake.

**Reply by jdeluyck** [reply]
The only thing I can add is that I used to get tons of warnings, and by adding `--bwlimit 8M:off` to the command line of rclone these have gone away, and it's generally more stable.
